src/components/layout/top-app-bar.tsx

```diff
@@ -4,31 +4,21 @@ import { useRouter } from "next/router";
 // Components
 import MaterialIcon from "@components/global/icon";
 
-// Data
-import { navGroups } from "@components/layout/pages";
-
 /**
  * Displays information and actions at the top of a screen
+ * @param title The display text, usually the page title
  */
-const TopAppBar = (): JSX.Element => {
-  const path = useRouter().asPath;
-
+const TopAppBar = ({ title }: { title: string }): JSX.Element => {
   return (
     <header className="w-full p-4 bg-light-surface2 dark:bg-dark-surface2 sm:p-7">
-      <button className="btn btn-text p-0">
+      <button className="btn btn-text p-0" onClick={() => history.back()}>
```

Comment:
are you trusting that the user will come in thing page via our GUI
what if they paste in the URL and when they click back they got sent back to chrome's new tab page
or the user click the URL from somewhere then it wont even do anything in that case